spec/intl.html

```diff
@@ -12,6 +12,10 @@
     The Intl object is not a function object. It does not have a [[Construct]] internal method; it is not possible to use the Intl object as a constructor with the *new* operator. The Intl object does not have a [[Call]] internal method; it is not possible to invoke the Intl object as a function.
   </p>
 
+  <p>
+    The Intl object has an internal slot, [[FallbackSymbol]], which is a new %Symbol% in the current realm.
```

Comment:
Ah, I misunderstood what you were trying to say (which is the root problem).  I thought you were saying that the "new  %Symbol%" was the key/name of the internal slot.  If the value is a realm specific intrinsic symbol, it probably needs to be added to Table 1 (of this spec).  Note that in Ecma-262 we don't have any realm-specific intrinsic symbols so rather than appearing in the corresponding Table 6 (Well known intrinsic objects) intrinsic symbols are listed in Ecma-262 Table 1 (Well known symbols).

We should probably also refer it the symbol as @@FallbackSymbol.

So, the spec statement could be: "The Intl object has an internal slot, [[FallbackSymbol]] whose value is initialized to  @@FallbackSymbol